project/test/suites/test_breadcrumbs_json.gd

```diff
@@ -26,7 +26,13 @@ func test_breadcrumbs_with_minimal_breadcrumb() -> void:
 		.end() \
 
 		.must_not_contain("type") \
-		.must_not_contain("data") \
+
+		.either() \
+			.must_not_contain("data") \
+		.or_else() \
+			.must_contain("data", {}) \
+		.end() \
```

Comment:
This test seems to accept both the presence and absence of certain fields, e.g. `data`. Does this depend on the platform? If so, it might be worth adding corresponding comments so we know what behavior to expect on each